CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -282,9 +229,67 @@ def add_item(self) -> DB_Response:
     def get_fridge_locations(self):
         pass
 
-class FrigeCheckIn(DB_Item):
-    def __init__(self, db_client: "botocore.client.DynamoDB"):
+
+class FridgeReport(DB_Item):
+
+    REQUIRED_FIELDS = ['fridge_username', 'status', 'notes', 'fridge_percentage']
+    ITEM_TYPES = {
+        'notes': 'S',
+        'fridge_username': 'S',
+        'image_url': 'S',
+        'timestamp': 'N',
+        'status': 'S',
+        'fridge_percentage': 'S'
+    }
+    TABLE_NAME = "fridge_report"
+    VALID_STATUS = set(["working", "needs cleaning", "needs servicing", "not at location"])
```

Comment:
Why is the list being explicitly cast to `set()`?

Reply:
It allows for quick lookups. 

```"needs cleaning" in VALID_STATUS``` is a O(1) lookup instead of an o(N) lookup if it was a list.

it's only a list of 4 so probably doesn't make a big different lol

Reply:
Yeah, even the list is `O(1)` since there isn't a lot of data. Instead of creating an array and then casting, let's just use `status = {element1, element2...}`?

Reply:
sure! I'm used to using set() because if you want to initialize an empty set you have to do set() instead of {}. But when you already have the data you can just do curly brackets I just realized :). I can update